_rules/6cbcbe.md

```diff
@@ -0,0 +1,206 @@
+---
+name: Links with identical accessible names serve equivalent purpose
+test_type: atomic
+
+description: |
+  This rule checks that identical accessible names are only used for links that serve an equivalent purpose
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of link, are [included in the accessibility tree](#included-in-the-accessibility-tree) and that have the same [non-empty](#non-empty) [accessible name](#accessible-name).
+
+### Expectation
+
+When activated, the links in each set of target elements resolve to resources that fulfill an [equivalent purpose](#equivalent-purpose) in relation to the [accessible names](#accessible-name) of the link.
+
+**Note:** Resolving the links includes potential redirects.
+
+## Assumptions
+
+* This rule assumes that the purpose of the links for links with identical names would not be ambiguous to users in general.
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- [Understanding Success Criterion 2.4.9: Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)
+- [URL parsing](#https://www.w3.org/TR/html52/infrastructure.html#parsing-urls)
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+Identical link text leads to identical URLs:
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+```
+
+#### Passed example 2
+
+Links resolves to same page after redirect:
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/redirect.html">Contact us</a>
+```
+
+#### Passed example 3
+
+Identical pages are located on different URLs:
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/index-copy.html">Contact us</a>
+```
+
+#### Passed example 4
+
+Same link text used for links going to pages where the content section is the same, but where the navigation options (bread crumbs and local sub menus) differ due to different placement in navigation hierarchy:
+
+```html
+<a href="/test-assets/6cbcbe/about/contact.html">Contact us</a>
+<a href="/test-assets/6cbcbe/careers/contact.html">Contact us</a>
+```
+
+#### Passed example 5
+
+URLs differ due to trailing slashes:
+
+```html
+<a href="/test-assets/6cbcbe/">Contact us</a> 
+<a href="/test-assets/6cbcbe">Contact us</a>
+```
+
+#### Passed example 6
+
+Pages contain different amounts of information and/or differently worded information, but fulfils same purpose in relation to the link:
```

Comment:
I don't get this one, isn't that like saying that two links to completely different contact pages for different department is okay?

Reply:
What if the link text is "call us", and the pages are the same as they are now? 
The point is, that as long as you still get the same phone number, they can fulfil the same purpose, whereas if you are actually calling different departments, it's not the same purpose anymore.

Reply:
Updated it to say "call us". @Brynanders, do you think it works now?